src/components/Comments/ReplyCommentItem.js

```diff
@@ -2,20 +2,22 @@
 import {useNavigation} from '@react-navigation/native';
 import * as React from 'react';
 import {StyleSheet, Text, TouchableOpacity, View} from 'react-native';
+import Toast from 'react-native-simple-toast';
 import IconEn from 'react-native-vector-icons/Entypo';
+import {IcDmAnon} from '../../assets/icons/ic_dm_anon';
 
 import MemoIc_arrow_down_vote_off from '../../assets/arrow/Ic_downvote_off';
 import MemoIc_downvote_on from '../../assets/arrow/Ic_downvote_on';
 import MemoIc_arrow_upvote_off from '../../assets/arrow/Ic_upvote_off';
 import MemoIc_upvote_on from '../../assets/arrow/Ic_upvote_on';
 import MemoCommentReply from '../../assets/icon/CommentReply';
 import MemoSendDM from '../../assets/icon/SendDM';
-import SendDMAnonBlock from '../../assets/icons/images/send-dm-anon-black.svg';
 import SendDMBlack from '../../assets/icons/images/send-dm-black.svg';
 import {Context} from '../../context';
 import useDMMessage from '../../hooks/core/chat/useDMMessage';
 import useCreateChat from '../../hooks/screen/useCreateChat';
 import ProfilePicture from '../../screens/ProfileScreen/elements/ProfilePicture';
+import {getAllowAnonDmStatus} from '../../service/chat';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statements have been modified to include new icons and functions. The `IcDmAnon` icon has been added, and the `getAllowAnonDmStatus` function has been imported. These changes seem necessary for the new feature being implemented.
```diff
+ import Toast from 'react-native-simple-toast';
+ import {IcDmAnon} from '../../assets/icons/ic_dm_anon';
+ import {getAllowAnonDmStatus} from '../../service/chat';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->